taskcluster/translations_taskgraph/transforms/find_upstreams.py

```diff
@@ -63,7 +63,7 @@ def get_cleaning_type(upstreams):
     candidates = set()
 
     for upstream in upstreams:
-        if upstream.kind not in ("bicleaner", "clean-corpus"):
+        if upstream.kind not in ("clean-corpus", "bicleaner"):
```

Comment:
Since you ended up not changing anything, can you put the order back so this doesn't show up in the diff?